app/assets/src/components/PhyloTreeList.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+class PhyloTreeList extends React.Component {
```

Comment:
We should probably have a component for "list views". Is this type of views used anywhere else already?

Reply:
not really. Well, Samples.jsx is one, but it's not very reusable in this case